src/CommitmentsV2/SFA.DAS.CommitmentsV2.Hashing.Temp/HashingService.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
```

Comment:
Why are we using a Hashing Service? Does this not have a nuget package?

Reply:
Note to pete:
HashingService is now .net standard compatible. You should update the sfa.das.hashing nuget to the latest version and then remove the HashingTemp project from the solution.
The name spaces will then need fixing up.